src/main/resources/assets/clientcommands/lang/en_us.json

```diff
@@ -16,6 +16,11 @@
   "commands.careastats.output.blocksMatched": "Matched %d out of %d total blocks",
   "commands.careastats.output.entitiesFound": "Found %d entities in this area",
 
+  "commands.cautoprefix.success": "Prefixing \"%s\" in all the messages you send from now on",
```

Comment:
nit: "Prefixing \"%s\" in all your following messages" is more consise.

Reply:
i prefer the current one, but maintainers are free to say other wise

Reply:
I like the current one better.